src/main/java/org/snomed/snowstorm/core/data/services/BranchMetadataHelper.java

```diff
@@ -19,6 +19,8 @@
 	// Metadata for internal use only and can't be updated via branch metadata REST service
 	public static final String INTERNAL_METADATA_KEY = "internal";
 
+	public static final String CLASSIFIED_KEY = "classified";
```

Comment:
I couldn't find a constant for classified, so I added it here beside what I also had to use.